tests/testthat/test_system.R

```diff
@@ -307,6 +307,24 @@ test_that("geocode_japan_prefecture", {
   expect_equal(FALSE, any(is.na(res$latitude)))
 })
 
+test_that("japan_city_code", {
+  # original: tibble(x=c("北海道", "東京都"), y=c("札幌市白石区", "稲城市"))  
```

Comment:
To avoid garbling of the code on Windows, we should avoid using Kanji even in comment.